package.json

```diff
@@ -6,9 +6,9 @@
   "typings": "index.d.ts",
   "scripts": {
     "preinstall": "node scripts/preinstall.js",
-    "install": "(node scripts/should_rebuild && node-gyp-build) || exit 0",
-    "rebuild": "node-gyp rebuild --jobs=max",
-    "prebuild": "node scripts/prebuild.js",
+    "install": "(node scripts/should_rebuild && node scripts/download_lib.js && node-gyp-build) || exit 0",
+    "rebuild": "node scripts/download_lib.js && node-gyp rebuild --jobs=max",
+    "prebuild": "node scripts/download_lib.js && node scripts/prebuild.js",
```

Comment:
Since I don't understand the scripts flow fully, I added the download script to these 3, but I'm really unsure about the `install` one. Does it make sense to you ?